osgp/platform/osgp-core/src/test/java/org/opensmartgridplatform/core/application/tasks/ScheduledTaskExecutorServiceTest.java

```diff
@@ -76,15 +76,17 @@ public class ScheduledTaskExecutorServiceTest {
    * @throws JobExecutionException
    */
   @Test
-  public void testRunFunctionalException()
+  void testRunFunctionalException()
       throws FunctionalException, UnknownHostException, JobExecutionException {
     final List<ScheduledTask> scheduledTasks = new ArrayList<>();
+    final Timestamp scheduledTime = new Timestamp(Calendar.getInstance().getTime().getTime());
```

Comment:
Have you considered using for example:
```suggestion
    final Timestamp scheduledTime = new Timestamp(Instant.now().toEpochMilli());
```
or
```suggestion
    final Timestamp scheduledTime = new Timestamp(System.currentTimeMillis());
```